src/components/filters/Filters.jsx

```diff
@@ -0,0 +1,197 @@
+import { useState, useEffect } from "react";
+import "./Filters.css";
+import { useProduct } from "../../contexts/product-context";
+
+export const Filters = () => {
+  const {state:filterState,dispatch:dispatchFilters} = useProduct();
```

Comment:
Ok so you are renaming it here itself as filterState and filterDispatch. Instead you can use it globally in the context with these names

Reply:
done